src/freenas/root/.login

```diff
@@ -2,7 +2,4 @@ if ( -f /usr/local/sbin/hactl) then
         /usr/local/sbin/hactl status -q
 endif
 
-echo
-echo "Warning: settings changed through the CLI are not written to"
-echo "the configuration database and will be reset on reboot."
-echo
+cat .warning
```

Comment:
I don't see `src/freenas/root/.warning`, is it generated somewhere else?

Reply:
I added in a second commit, your page was probably outdated